.github/workflows/test-phpunit.yml

```diff
@@ -40,13 +40,15 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        php-versions: ['7.3', '7.4', '8.0']
+        php-versions: ['7.3', '7.4', '8.0', '8.1']
         db-platforms: ['MySQLi', 'Postgre', 'SQLite3', 'SQLSRV']
         mysql-versions: ['5.7']
         include:
-          - php-versions: 7.4
+          - php-versions: '7.4'
             db-platforms: MySQLi
-            mysql-versions: 8.0
+            mysql-versions: '8.0'
+          - php-versions: '8.1'
+            composer-flag: '--ignore-platform-req=php'
```

Comment:
Is this necessary for some of our dependents? I think the framework should work fine without it, no?

Reply:
Some of our dev-deps expressly forbids 8.1 yet. This can be lifted once all our deps allows 8.1